assets/Dockerfile

```diff
@@ -5,9 +5,10 @@ COPY --from=python:3.6.13-alpine3.13 /usr/local/ /usr/local/
 COPY --from=python:3.7.10-alpine3.13 /usr/local/ /usr/local/
 COPY --from=python:3.8.8-alpine3.13 /usr/local/ /usr/local/
 COPY --from=python:3.9.2-alpine3.13 /usr/local/ /usr/local/
+COPY --from=python:3.10.4-alpine3.14 /usr/local/ /usr/local/
```

Comment:
[ask] There's no alpine3.13 images of Python 3.10 on Docker Hub.
Which image should I use here?

https://hub.docker.com/_/python?tab=tags

Reply:
It is ok to use alpine 3.14. Can you update all images to 3.14?

Reply:
OK!

Reply:
done in f698192570aa44ab24565d7ab653d1ea57b498ce